api/v1/openapi.yaml

```diff
@@ -249,6 +249,29 @@ paths:
           description: Endpoints with provided parameters not found
         '429':
           description: Rate-limiting too many requests in the given time frame
+    delete:
+      summary: Deletes a list of endpoints
+      description: |
+        Deletes a list of endpoints that have endpoints matching the provided properties
```

Comment:
How does a user know what the currently supported properties are?

Edit: Adding a reference to the below `EndpointDeleteRequest` field should help.

Reply:
This is already referenced in the `parameters` section below, ref. https://github.com/cilium/cilium/blob/da64b11ce28ce83f95b6221ca11d50991d635ac4/api/v1/openapi.yaml#L258-L259

It shows up as follows in the Swagger editor documentation:

![image](https://github.com/cilium/cilium/assets/539708/7203d71a-c512-4a5f-9cb9-2e647361b3c0)

Reply:
Oh nice! TIL about the online editor.